portfolio/src/constants.ts

```diff
@@ -0,0 +1,133 @@
+import AmazonNavigatorImgSrc from "./assets/Project1.png";
+import PlantParenthoodImgSrc from "./assets/Project2.png";
+import PasswordGeneratorImgSrc from "./assets/Project3.png";
+import TextEditorImgSrc from "./assets/Project4.png";
+import { ExperienceLevel } from "./types";
+
+export const headerOptions = [
+  {
+    name: "Projects",
+    href: "#projects",
+  },
+  {
+    name: "Experience",
+    href: "#experience",
+  },
+  {
+    name: "About Me",
+    href: "#about-me",
+  },
+  {
+    name: "Skills",
+    href: "#skills",
+  },
+  {
+    name: "Contact",
+    href: "#contact",
+  },
+];
+
+export const projects = [
+  {
+    id: "amazonNavigator",
+    imgSrc: AmazonNavigatorImgSrc,
+    githubHref: "https://github.com/alla0810/Group9",
+    livePreviewHref: "https://alla0810.github.io/AmazonNavigator/",
+    description:
+      "The AMAZON NAVIGATOR is designed for customers to view item reviews and pricing.",
+  },
+  {
+    id: "plantParenthood",
+    imgSrc: PlantParenthoodImgSrc,
+    githubHref: "https://github.com/VictoriaSweet/scary-potato",
+    livePreviewHref: "https://nameless-wave-14504-8d3cae4aaa8e.herokuapp.com/",
+    description:
+      "Plant Parenthood is the application to help the those who are new to the hobby or just maybe even a weekend plant-sitter or award winning growers.",
+  },
+  {
+    id: "passwordGenerator",
+    imgSrc: PasswordGeneratorImgSrc,
+    githubHref: "https://github.com/tfkjosh/Password-generator",
+    livePreviewHref: "https://tfkjosh.github.io/Password-generator/",
+    description:
+      "This application is designed for an employee to generate a random password based on criteria they've selected.",
+  },
+  {
+    id: "textEditor",
+    imgSrc: TextEditorImgSrc,
+    githubHref: "https://github.com/tfkjosh/PWA-Challenge",
+    livePreviewHref: "https://pwa-challenge1-b508091ee337.herokuapp.com/",
+    description:
+      "This application features a number of data persistence techniques, insuring the application runs regardless of browser supported function.",
+  },
+];
+
+interface TechnologySection {
+  name: string;
+  level: ExperienceLevel;
+}
```

Comment:
A simple `interface` for the shape of each list element in the `technologies` list defined below. Here you can see that we are using that `ExperienceLevel` string literal type to define `level` as a discrete value of either `Beginner`, `Intermediate`, or `Advanced`.